packages/components/src/Slider/Slider.snap.test.js

```diff
@@ -63,5 +63,14 @@ describe('Slider component', () => {
 			// then
 			expect(result).toBe(2);
 		});
+
+		it('should return -1 when value is null', () => {
```

Comment:
I think that this kind of tests has nothing to do in `snapshot` tests file. `FilterBar`, for example, has a `FilterBar.snapshot.test.js` for snapshot tests and a `FilterBar.test.js` for unit tests. Could you move it ?